lib/strategy.js

```diff
@@ -61,7 +61,7 @@ function Strategy(options, verify) {
 
   OAuth2Strategy.call(this, options, verify);
   this.name = 'onshape';
-  this._userProfileURL = options.userProfileURL || 'https://cad.onshape.com/api/users/session';
+  this._userProfileURL = options.userProfileURL || 'https://cad.onshape.com/api/users/sessioninfo';
```

Comment:
Can you look around the documentation in passport-onshape and see if there's a reference to `options.userProfileURL` that also needs to be updated?